LeadKit/LeadKit/Extensions/CGContext/CGContext+Initializers.swift

```diff
@@ -1,33 +1,46 @@
 //
-//  CGContext+Initializers.swift
-//  LeadKit
+//  Copyright (c) 2017 Touch Instinct
 //
-//  Created by Ivan Smolin on 04/10/16.
-//  Copyright © 2016 Touch Instinct. All rights reserved.
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
 //
 
 import CoreGraphics
 
 public extension CGContext {
 
     /**
-     method which creates an instance of CGContext with parameters taken from a given image
+     creates an instance of CGContext with parameters taken from a given image
 
      - parameter forCGImage: CGImage instance from which the parameters will be taken
      - parameter fallbackColorSpace: fallback color space if image doesn't have it
      */
-    public static func create(forCGImage cgImage: CGImage,
-                              fallbackColorSpace: CGColorSpace = CGColorSpaceCreateDeviceRGB()) -> CGContext? {
-
-        return create(width: cgImage.width,
-                      height: cgImage.height,
-                      bitmapInfo: cgImage.bitmapInfo,
-                      colorSpace: cgImage.colorSpace ?? fallbackColorSpace,
-                      bitsPerComponent: cgImage.bitsPerComponent)
+    public convenience init?(forCGImage cgImage: CGImage,
+                             fallbackColorSpace: CGColorSpace = CGColorSpaceCreateDeviceRGB()) {
+        self.init(width: cgImage.width,
```

Comment:
https://developer.apple.com/library/content/documentation/Swift/Conceptual/Swift_Programming_Language/Initialization.html
can't be omitted in convenience init =(